server/controllers/mealInfo.js

```diff
@@ -0,0 +1,8 @@
+const { getMealInfo } = require('../database/getQueries/getMealInfo');
+
+exports.get = (req, res) => {
+  const { mealId } = req.params;
+  getMealInfo(mealId)
+    .then(row => res.send(row))
+    .catch(() => res.send('Could not find meal'));
```

Comment:
i would suggest adding `.status(500)` to the res as in `res.status(500).send(something)` so it goes straight to the reject instead of having an if statement in the resolved promise in the frontend